src/config.rs

```diff
@@ -286,7 +291,7 @@ impl Config {
                         Value::String(s) => {
                             let extension = Path::new(&s).extension();
                             if extension.is_some() && extension.unwrap() == "jar" {
-                                config.apktool_file = s.clone();
+                                config.apktool_file = PathBuf::from(s.clone());
```

Comment:
We don't actually need to clone here, right? we can use the reference and avoid allocating.